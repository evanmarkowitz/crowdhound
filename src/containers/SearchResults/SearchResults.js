import React, { Component } from './node_modules/react'

export class SearchResults extends Component {
  render() {
    return (
      <div>
        
      </div>
    )
  }
}

export default SearchResults;