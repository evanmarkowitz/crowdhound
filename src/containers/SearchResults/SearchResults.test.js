import React from 'react';
import {SearchResults, GET_ALL_DOG_QUERY} from './SearchResults';
import {MockedProvider} from '@apollo/react-testing'

import { shallow } from 'enzyme';



describe('SearchResults', () => {
  let wrapper;
  let mocks;
  beforeEach(() => {
     

      mocks = [
      {
        request: {
          query: GET_ALL_DOG_QUERY,

        },
        result: {
          data: {
            dog: { id: '1',  photos: {sourceUrl: ''} },
          },
        },
      },
    ];
    wrapper = shallow(
      <MockedProvider mocks={mocks} addTypename={false}>
        <SearchResults />
      </MockedProvider>
    );

  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

})
