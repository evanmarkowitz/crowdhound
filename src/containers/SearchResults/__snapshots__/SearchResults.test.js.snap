// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults should map the snapshot 1`] = `ShallowWrapper {}`;

